JSDemos/Demos/Toolbar/Multiline/description.md

```diff
@@ -1,5 +1,5 @@
-## Configure Toolbar Items
+This demo demonstrates the multiline mode in the Toolbar component. To see step-by-step instructions on how to get started with the DevExtreme Toolbar component, refer to the following tutorial: [Getting Started with Toolbar](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/). 
 
-## Specify Item Location
+In this demo, drag the handle of the component container on the right to resize the Toolbar. If you enable the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property, the Toolbar is displayed in multiple lines after resize. The number of lines is restricted by the minimum width of the container.
```

Comment:
Again, I'm not sure about the last sentence. As I understand, there's really no limitation on the number of lines. It depends on the container width and also on the number of items in the toolbar. So it could be anything - I don't think we need to talk about "restrictions". 

I'm also trying to simplify the text a bit. One thing I wanted to note is the beginning. If you write specific instructions - "drag the handle" - then you don't need to say "In this demo". You're asking the reader to do something right now, so it's clear that you talk about this page. But I guess you need those words as a transition from the previous sentences back into this specific demo.... That's why I changed it to "you can drag" instead of just "drag". 

```suggestion
In this demo, you can drag the Toolbar container's resize handle. If you enable the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property, you'll see how the Toolbar wraps onto multiple lines. 
```